[2025-06-19T12:16:52.416+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: week_3_weather_api_dag.create_weather_table manual__2025-06-19T12:16:49.824208+00:00 [queued]>
[2025-06-19T12:16:52.426+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: week_3_weather_api_dag.create_weather_table manual__2025-06-19T12:16:49.824208+00:00 [queued]>
[2025-06-19T12:16:52.427+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 1
[2025-06-19T12:16:52.442+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): create_weather_table> on 2025-06-19 12:16:49.824208+00:00
[2025-06-19T12:16:52.450+0000] {standard_task_runner.py:60} INFO - Started process 88 to run task
[2025-06-19T12:16:52.457+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'week_3_weather_api_dag', 'create_weather_table', 'manual__2025-06-19T12:16:49.824208+00:00', '--job-id', '41', '--raw', '--subdir', 'DAGS_FOLDER/week_3_weather_api_dag.py', '--cfg-path', '/tmp/tmpjbrx5z2j']
[2025-06-19T12:16:52.458+0000] {standard_task_runner.py:88} INFO - Job 41: Subtask create_weather_table
[2025-06-19T12:16:52.515+0000] {task_command.py:423} INFO - Running <TaskInstance: week_3_weather_api_dag.create_weather_table manual__2025-06-19T12:16:49.824208+00:00 [running]> on host b96384fab38e
[2025-06-19T12:16:52.823+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='week_3_weather_api_dag' AIRFLOW_CTX_TASK_ID='create_weather_table' AIRFLOW_CTX_EXECUTION_DATE='2025-06-19T12:16:49.824208+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-19T12:16:49.824208+00:00'
[2025-06-19T12:16:52.860+0000] {base.py:83} INFO - Using connection ID 'weather_***_conn' for task execution.
[2025-06-19T12:16:53.008+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-06-19T12:16:53.021+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=week_3_weather_api_dag, task_id=create_weather_table, execution_date=20250619T121649, start_date=20250619T121652, end_date=20250619T121653
[2025-06-19T12:16:53.063+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-06-19T12:16:53.113+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
